### ISSUE TITLE
Subscribe or Activation Window (and Subscriber) redirects incorrectly

### ISSUE THREAD

**Issue by JLewis12** [mkoza178/PJACS-Implemenation#13]
Where is are these controlled at?  I couldn’t locate in CMS and  It is pointing to dsssubsciber & dsssubscriber Syracuse currently and not the presentation landing page. 

I believe there was talk in the past about removing the Zip entry field to have the “View Offers” button go directly to the SP landing pages to enter Zip’s.

![image](https://user-images.githubusercontent.com/26151269/39925610-2e2962bc-54e1-11e8-9c6e-b60504dc7ed4.png)

**Comment by mkoza178** [comment]
It's set here in newspaper attributes:
https://prnt.sc/jgl7il

We agreed to not remove the view offers box.

**Comment by JLewis12** [comment]
Okay your screen shot shows it as inactive? Is this the location to set the redirect URL to the presentation panel?

**Comment by mkoza178** [comment]
Right, turn it on and set the subscription panel URL and you'll be all set!

**Comment by JLewis12** [comment]
Okay - i'll take a look.  Not sure why it would be "off" and still functioning. but it's your software :-)

**Comment by mkoza178** [comment]
Well, I can tell you:

If no value is set, we default to the dssSub page.

**Comment by JLewis12** [comment]
Got it..

**Comment by mkoza178** [comment]
You're under Attributes. You need to be under AttributeValues.

It controls both.

**Comment by JLewis12** [comment]
NM - I was not in values

**Comment by JLewis12** [comment]
Yep - made the change and added (https://test-subscribe.nj.com/Advance_Standard?SiteCode=SL)  now we get nothing on either link..  ?
![image](https://user-images.githubusercontent.com/26151269/39932029-dec5494e-54f3-11e8-8947-0a38581748ea.png)

**Comment by mkoza178** [comment]
https://test-**subscriber**.nj.com/Advance_Standard?SiteCode=SL it says 'Subscriber' in the url. 
Cleared cache in CMS and then tried incognito and it works for me now!

**Comment by JLewis12** [comment]
Roger that - confirmed....  Thanks
